### ISSUE TITLE
Duplicated text in the shipAreaFromMass() and shipWaterForce() functions description

### ISSUE THREAD

**Issue by alexee-mayette** [dankelley/whalestrike#26]
In both pages for the [shipAreaFromMass()](https://dankelley.github.io/whalestrike/reference/shipAreaFromMass.html) and [shipWaterForce()](https://dankelley.github.io/whalestrike/reference/shipWaterForce.html), the text for the description is duplicated.

**Comment by dankelley** [comment]
Oh, that's an interesting error.  It looks like several items are duplicated.  I will look into it.

**Comment by dankelley** [comment]
Thanks.  This is fixed in the "main" branch of 
 https://github.com/dankelley/whalestrike 
in commit  98701aa997784dfe99fd92908c9a466464f5eb82.

PS. it arose because I had defined these functions twice.  Originally, all the code was in a single R file, but when I decided to open up the code so more people could use it, I realized that it would help to break the code up some more, in hopes that users could suggest new additions more easily.  Long story short, I copied the code to a new file without deleting it from the original.  A silly mistake, I know.
